File: TestApiRejseplanen/TestApiRejseplanen/Model/Destination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TestApiRejseplanen.Model
{
    public class Destination
    {
        public string Name { get; set; }
        public string Type { get; set; }
        public int RouteIdx { get; set; }
        public string Time { get; set; }
        public string Date { get; set; }

        public Destination(string name, string type, int routeIdx, string time, string date)
        {
            Name = name;
            Type = type;
            RouteIdx = routeIdx;
            Time = time;
            Date = date;
        }
    }
}

File: Webpack/src/js/destination.ts
export class Destination {
    public Name: string;
    public Type: string;
    public RouteIdx: number;
    public Time: string;
    public RtTime: string;
    public Date: string;
    public RtDate: string;
    public Track: string;
    public RtTrack: string;

    constructor(name: string, type: string, time: string, date: string,
                routeIdx?: number, track?: string, rtTime?: string, rtDate?: string, rtTrack?: string ) {
    this.Name = name;
    this.Type = type;
    this.Time = time;
    this.Date = date;

    this.RouteIdx = routeIdx;
    this.Track = track;
    this.RtTime = rtTime;
    this.RtDate = rtDate;
    this.RtTrack = rtTrack;
    }
}
